lib/standard/merges_settings.rb

```diff
@@ -41,8 +41,11 @@ def determine_command(argv)
         :help
       elsif (argv & ["--version", "-v"]).any?
         :version
+      elsif (argv & ["--gen-ignore"]).any?
```

Comment:
Is `--gen-ignore` the exact arg name rubocop uses? It's not clear at all to me what this flag actually does if I'm not literally thinking about this feature. I'd rather it was named something longer and intent-revealing `--generate-ignore-todo-file`

Reply:
How does `--generate-todo` sound?